templates/list.hbs

```diff
@@ -4,39 +4,41 @@
 
   <div class="component__widget list__widget">
 
-    <div class="list__container{{#if _orderedList}} is-ordered-list{{/if}}" role="list">
+    <div class="list__container{{#if _orderedList}} is-ordered-list{{/if}}{{#if _columns}} has-columns{{/if}}" role="list">
```

Comment:
Suggest only applying the column styling if `_columns` is set to be greater than `1`?
```hbs
<div class="list__container{{#if _orderedList}} is-ordered-list{{/if}}{{#compare _columns '>' 1}} has-columns{{/compare}}" role="list">
```

Think that's the correct use of the helper; might need a test.